src/commands/fizzbuzz.js

```diff
@@ -0,0 +1,59 @@
+const Aquarius = require('../aquarius');
+const fetch = require('node-fetch');
+
+class FizzBuzz extends Aquarius.Command {
+  constructor() {
+    super();
+
+    this.description = 'A fresh take on a classic programming problem.';
+  }
+
+  helpMessage(nickname) {
+    let msg = super.helpMessage();
+
+    msg += 'Usage:\n';
+    msg += `\`\`\`@${nickname} fizzbuzz [number]\`\`\``;
+    return msg;
+  }
+
+  static makeFizzBuzzCall(number) {
+    return fetch(`https://fozzbizz.herokuapp.com/fizzbuzz?num=${number}`)
+      .then(response => {
+        if (response.ok) {
+          return response.json();
+        }
+
+        throw response.statusText;
+      }).then(fizzBuzzJSON => {
+        if (fizzBuzzJSON.is_fizz && fizzBuzzJSON.is_buzz) {
```

Comment:
if I do that I get `falseBuzz` and `Fizzfalse`

But I put a ternary operator instead.